cmd/kpod/inspect.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"text/template"
+
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+const (
+	defaultFormat = `Container: {{.Container}}
+ID: {{.ContainerID}}
+`
+	inspectTypeContainer = "container"
+	inspectTypeImage     = "image"
+	inspectAll           = "all"
+)
+
+var (
+	inspectFlags = []cli.Flag{
+		cli.StringFlag{
+			Name:  "type, t",
+			Usage: "look at the item of the specified `type` (container or image) and name",
+		},
+		cli.StringFlag{
+			Name:  "format, f",
+			Usage: "use `format` as a Go template to format the output",
+		},
+		cli.BoolFlag{
+			Name:  "size",
+			Usage: "display total file size if the type is container",
+		},
+	}
+	inspectDescription = "Displays a build container's or built image's configuration."
+	inspectCommand     = cli.Command{
+		Name:        "inspect",
+		Usage:       "Displays the configuration of a container or image",
+		Description: inspectDescription,
```

Comment:
Can you get a better description, perhaps multi-line to not just be a restatement of the Usage line.

Perhaps steal something from the man page.